.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -9,6 +9,15 @@ assignees: ''
 
 <!--
 
+!!!!!IF YOU ARE A TROLL READ THIHS FIST!!!!
```

Comment:
THIHS? FIST? 🤔

Reply:
I hate these kind of typoes!

Reply:
me to :sob:

Reply:
I'm gonna need to make a pull request fixing that typo.